### ISSUE TITLE
Investigate if discrete slider should have a "dropless" indicator variant for use with long numbers

### ISSUE THREAD

**Issue by kfranqueiro** [material-components/material-components-web#1437]
We should implement another style variant for discrete sliders that just shows an appropriately-colored label without the drop, for use in cases where the maximum is large (e.g. 200+) or where the step value is not an integer (e.g. .5).

Codepen demonstrating the problem: https://codepen.io/kfranqueiro/pen/jGXNrK

**Comment by berenddeboer** [comment]
Definitely, I really need this. And also for cases where you present the value on some other section of the screen.
